### PR TITLE
feat(banner): improve banner with perseverance design

### PR DESCRIPTION
**Pull request by gcusnieux**
https://graviteecommunity.atlassian.net/browse/APIM-257

---
📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-hjgvvrimfc.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-e8d4fe0
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-e8d4fe0
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-e8d4fe0
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-e8d4fe0
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-7c2515c
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-7c2515c
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-7c2515c
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-7c2515c
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-1d2d388
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-1d2d388
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-1d2d388
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-1d2d388
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-d175bde
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-d175bde
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-d175bde
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-d175bde
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-f00774c
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-f00774c
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-f00774c
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-f00774c
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-0cca2bb
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-0cca2bb
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-0cca2bb
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-0cca2bb
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-4caefbc
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-4caefbc
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss [hunk 1]
```diff
@@ -38,40 +38,64 @@ $types: (
...
-    @include mat.typography-level($typography, body-1);
+    &__title {
+      display: flex;
+      flex: 1;
+      align-items: center;
+      padding: 4px 0;
+    }
+
+    &__action {
+      align-self: flex-start;
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss:90]: the button does not centre and remain aligned at the top? 
image

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-5cf3594
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-5cf3594
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-5cf3594
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-5cf3594
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.8.1-improve-banner-8929a74
# or
yarn add @gravitee/ui-particles-angular@4.8.1-improve-banner-8929a74
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-8929a74
# or
yarn add @gravitee/ui-policy-studio-angular@4.8.1-improve-banner-8929a74
```

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**